### ISSUE TITLE
How far can a mali gpu go with box4droid?

### ISSUE THREAD

**Issue by Ghost-mory-021** [Herick75/Box4Droid#17]
I know mali gpu s are not supported at the moment but i wanted to know if the not supported type is that you can't run games at all ... Or you can't even get in the wine ?
I done everything correctly the first app is showing connected right as the tutorial video everything is fine but x11 app is still not connected 
is it just like it in all mali gpu or I'm doing something wrong?
![Screenshot_2023-04-19-23-58-25-914_com termux x11](https://user-images.githubusercontent.com/131301963/233194652-f6e0bca6-090a-4cce-acc6-7cd965b99e33.jpg)

**Comment by Ghost-mory-021** [comment]
Yes i run x11 and tryed all resolution options but no Matter what the x11 never connect

**Comment by Ilya114** [comment]
Try:
Xvfb :1 -ac &
DISPLAY=:1 termux-x11 &
